tests/tests.h

```diff
@@ -187,6 +189,51 @@ namespace Testing
 
 
 
+// Construct a random point, with coordinates between min and max
+template<int dim>
+inline Point<dim> random_point(const double &min=0.0,
+                               const double &max=1.0)
+{
+  Assert(max >= min, ExcMessage("Make sure max>=min"));
+  Point<dim> p;
+  for (unsigned int i=0; i<dim; ++i)
+    p[i] = min+(Testing::rand()*(max-min))/((double)RAND_MAX);
+  return p;
+}
+
+
+
+// Get a positive random value between min and max
+template<type T=double>
+T random_value(const T &min, const T &max)
```

Comment:
why not give defaults `min=0, max=1` and remove another function below?

Reply:
because that takes only the maximum value. This allows one to use

~~~
random_value(50); // a uniform integer between 0 and 50
random_value(1.0); // a uniform double between 0 and 1.0
random_value(-1.0, 1.0); // a uniform double between -1 and 1
~~~

Reply:
just switch the order and do 
```
random_value(const T &max=1., const T &min=0.)
```
and you have a single function to cover all the cases in the above.